src/hist_smooth/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .grid_plot_hist import plot_grid_with_smoothing
+from .smoothing_api import Smoothing

```

Comment:
```suggestion
from hist_smooth.grid_plot_hist import plot_grid_with_smoothing
from hist_smooth.smoothing_api import Smoothing
```

I'd recommend using absolute imports (like in my suggested change) instead of relative imports (using the directory structure as it is now) as this makes the code easier to read in the long run and to maintain. So I'd make these changes everywhere.